tests/loader_get_proc_addr_tests.cpp

```diff
@@ -91,9 +98,15 @@ TEST(GetProcAddr, GlobalFunctions) {
         handle_assert_has_value(CreateInstance);
     }
     {
+#if !defined(VULKANSC)
         // Create a 1.3 instance - now everything should return NULL
         InstWrapper inst{env.vulkan_functions};
         inst.create_info.api_version = VK_MAKE_API_VERSION(0, 1, 3, 0);
+#else
+        // Create a VKSC 1.1 instance - now everything should return NULL
```

Comment:
1.1 again - is this supposed to be a version that doesn't exist here?

Reply:
Yep. This might be confusing as VK 1.3 is available at this point so this source is probably out of date compared to the ToT standard VK loader test source. I.E. at ToT VK loader source, I imagine this test has been updated such that is says something like, "Create a 1.4 instance - now everything should return null".

This portion of the test is intentionally trying to use a version of VKSC that doesn't exist in order to do some negative testing, so I used VKSC 1.1.

Reply:
👍